resolver/uuidResolver.go

```diff
@@ -0,0 +1,77 @@
+package resolver
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/Financial-Times/uuid-utils-go"
+	"time"
+)
+
+const dateTimeFormat = "2006-01-02T15:04:05.000Z0700"
+
+type UuidsAndDateResolver interface {
+	Resolve(reqData []byte, respData []byte) (*UuidsAndDate, error)
+}
+
+type UuidsAndDate struct {
+	UuidArr      []string
+	LastModified string
+}
+
+type fromRequestResolver struct {
+}
+
+func NewUuidResolver() UuidsAndDateResolver {
+	return &fromRequestResolver{}
+}
+
+func (r *fromRequestResolver) Resolve(reqData []byte, respData []byte) (*UuidsAndDate, error) {
+	cc := &contentCollection{}
```

Comment:
would be more readable to have: `cc := contentCollection{}`; and `err := json.Unmarshal(reqData, &cc)`